src/functions.js

```diff
@@ -0,0 +1,138 @@
+// select variables
+const todoListElement = document.getElementById('todo');
+let todos = JSON.parse(localStorage.getItem('todos')) || [];
```

Comment:
const for variables that do not need to be reassigned

Reply:
Thanks alot, noted 👍